MorphicServer/Endpoint.cs

```diff
@@ -122,17 +156,35 @@ public static async Task Run<T>(HttpContext context) where T: Endpoint, new()
                         {
                             await call();
                         }
+                        if (path != "/alive" && path != "/ready")
```

Comment:
Is there a way to do this with attributes on the endpoint or method so we don't have to hard-code paths here?

Reply:
Pushed a new change. That was surprisingly easy. Let me know if that's the right way to do it.